github-merge-flow.jsonc

```diff
@@ -0,0 +1,20 @@
+// IMPORTANT: This file is read by the merge flow from main branch only. 
+{
+    "merge-flow-configurations": {
+        // Automate merging runtime release/6.0 branch into release/6.0-staging
+        "release/6.0":{
+            "MergeToBranch": "release/6.0-staging",
+            "ExtraSwitches": "-QuietComments"
+        },
+        // Automate merging runtime release/8.0 branch into release/8.0-staging
+        "release/8.0":{
+            "MergeToBranch": "release/8.0-staging",
+            "ExtraSwitches": "-QuietComments"
+        },
+        // Automate merging runtime release/8.0-rc branches back to release/8.0
+        "release/8.0-rc":{
+            "MergeToBranch": "release/8.0",
```

Comment:
Shouldn't this be 9.0-rc?

Reply:
Since we don't yet have release/9.0 or release/9.0-rc* branches, I think we're good for now to merge so we don't get blocked on Monday, but I suspect we need these lines updated.

Reply:
The configuration file was prepared based on the https://github.com/dotnet/versions/blob/main/Maestro/subscriptions.json file. 
If there will be a need for the new onboarding branch, please use current (from the PR) file to update the configuration for additional branches in the merge flow chain :)